### PR TITLE
Holidays Luxembourg

### PR DESCRIPTION
**Pull request by placid2000**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module github.com/placid2000/cal/v2
```
Comment (rickar) [inline go.mod:1]: Please remove this file. There is already a go.mod in the v2 directory.

File: v2/lu/lu_holidays.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
+// (c) Rick Arnold. Licensed under the BSD license (see LICENSE).
+
+// Package be provides holiday definitions for Luxembourg.
+package be
```
Comment (rickar) [inline v2/lu/lu_holidays.go:4]: Shouldn't the package be lu?

File: v2/lu/lu_holidays_test.go [hunk 3]
```diff
@@ -0,0 +1,123 @@
+// (c) Rick Arnold. Licensed under the BSD license (see LICENSE).
+
+package be
```
Comment (rickar) [inline v2/lu/lu_holidays_test.go:3]: Same here

File: v2/lu/lu_holidays.go (hunk 2 above)
Comment (placid2000) [reply v2/lu/lu_holidays.go]: Correct. Thanks for the hint

**Comment by rickar** [comment]
Thank you!

**Pull request merged**